src/objects/Tile.java

```diff
@@ -0,0 +1,72 @@
+package objects;
+
+public class Tile {
+	private String type;
+	private String item;
+	private boolean moveable;
```

Comment:
It would be nice if there was a boolean move[4] to check if you could visit adjacent tiles from the current one.

Reply:
Also forgot to mention, we're keeping resource files in their own folder, res, which is on the same level of src. Not sure how reading files with Scanner works, but if you need a solution, I have something that'll work.

Reply:
Just noticed another thing, are we going with a 0-indexed array for the gameboard?